docs/functions_for_comp.md

```diff
@@ -163,29 +147,34 @@ For example `select array_join([10,20]) as v, 'text' as t` will get 2 rows
 
 ### array_max
 
-`array_max(func, array)` apply the function to every element in the array and then returns the maximum value e.g. `array_max(x->x*x,[1,2])`returns 4
+**Overload 1:** `array_max(array)` returns the maximum value in the array. e.g. `array_max([1,2,3])` returns 3
 
-### array_min
+**Overload 2:** `array_max(func, array)` apply the function to every element in the array and then returns the maximum value e.g. `array_max(x->x*x,[1,2])`returns 4
 
-`array_min(func, array)` apply the function to every element in the array and then returns the minimum value e.g. `array_min(x->x*x,[1,2])`returns 1
+### array_min
 
-### array_sort
+**Overload 1:** `array_min(array)` returns the minimum value in the array. e.g. `array_min([1,2,3])` returns 1
 
-`array_sort(func, array)` sorts the array elements in ascending order. e.g. `array_sort([3,2,5,4])` returns [2,3,4,5]. You can pass a lambda function to it as the first argument to apply the function before the sort, e.g. `array_sort(x->-x,[3,2,5,4])`returns [5,4,3,2]
+**Overload 2:** `array_min(func, array)` apply the function to every element in the array and then returns the minimum value e.g. `array_min(x->x*x,[1,2])`returns 1
 
+### array_sort
 
+`array_sort([func,] array)` sorts the array elements in ascending order. e.g. `array_sort([3,2,5,4])` returns [2,3,4,5]. You can pass a lambda function to it as the first argument to apply the function before the sort, e.g. `array_sort(x->-x,[3,2,5,4])`returns [5,4,3,2]
 
 ### array_sum
 
 `array_sum([func,] array)` returns the sum value in the array. For example, `array_sum([2,6])` return 8. You can pass a lambda function to it as the first argument to apply on each element before calculating the sum, such as `array_sum(x->x*x,[2,6])` to get the sum for 2*2 and 6\*6, which is 40.
 
 ### array_fold
+
 `array_fold(lambda_function, arr1, arr2, ..., accumulator)` applies a lambda function to one or more equally-sized arrays and collects the result in an accumulator.
 
 For example, `array_fold(acc, x -> acc + (x * 2), [1, 2, 3, 4], to_int64(3))` will initialize the accumulator `acc` as value 3, then for each element in the `[1, 2, 3, 4]` array, assign it to `x` and accumulate them together with `x*2`. So the end result is 3+1*2+2*2+3*2+4*2=23.
 
 Make sure the return type of lambda function must be the same as the accumulator type.
 
+## Maps
```

Comment:
Since you added the h2 headings for map, etc, probably need to add h2 at very beginning for Arrays. The preview site https://deploy-preview-308--elastic-elion-a958b6.netlify.app/functions_for_comp